dat/Microsoft - MSX2.dat

```diff
@@ -1,7 +1,7 @@
 clrmamepro (
-	name "Microsoft - MSX2"
-	description "Microsoft - MSX2"
-	version 20130225-030822
+	name "Microsoft - MSX 2"
+	description "Microsoft - MSX 2"

```

Comment:
Here is another change where we want to deviate from what No-Intro provides... Moving the override to a different DAT override or something, would benefit us.